src/YMenuButton.cc

```diff
@@ -264,3 +264,36 @@ YMenuButton::getProperty( const string & propertyName )
 	return YWidget::getProperty( propertyName );
     }
 }
+
+YMenuItem *
+YMenuButton::findItem( std::vector<std::string> & path ) const
+{
+    return findItem( path.begin(), path.end(), itemsBegin(), itemsEnd());
+}
+
+YMenuItem *
+YMenuButton::findItem( std::vector<std::string>::iterator path_begin,
+                       std::vector<std::string>::iterator path_end,
+                       YItemConstIterator begin,
+                       YItemConstIterator end ) const
+{
+    for ( YItemConstIterator it = begin; it != end; ++it )
+    {
+        YMenuItem * item = dynamic_cast<YMenuItem *>(*it);
+        // Test that dynamic_cast didn't fail
+        if ( !item ) return nullptr;
+
+        if( item->label() == *path_begin ) {
```

Comment:
Please use the coding style that is used everywhere else in libyui packages: Opening braces _always_ on the new line.

Years ago, I summarized my reasoning here (different project, but same coding style and same reasoning):

  https://github.com/shundhammer/qdirstat/blob/master/doc/Contributing.md

Back when we wrote libyui we did have a mandatory C++ coding style, and I took great pains to do it consistently throughout all my libyui packages.

Reply:
Point noted, will fix other cases, which I can find. Cheers.